server/sonar-web/src/main/js/apps/coding-rules/templates/rule/coding-rules-rule-profile.hbs

```diff
@@ -55,32 +55,28 @@
   {{#if actions.edit}}
     {{#unless isBuiltIn}}
       <td class="coding-rules-detail-quality-profile-actions">
-        <div class="button-group">
```

Comment:
I noticed that in some case we have a border under the buttons and sometimes not : 
![image](https://user-images.githubusercontent.com/1725611/32228857-b232ad94-be4f-11e7-8d08-7410eb68e1bd.png)

Another one : 
![image](https://user-images.githubusercontent.com/1725611/32228788-86ef2784-be4f-11e7-885f-ed427d638591.png)